docs/ref/templates/api.txt

```diff
@@ -139,6 +139,13 @@ overridden by what's passed by
       Tags and filters from built-in libraries can be used without first calling
       the :ttag:`{% load %}<load>` tag.
 
+    * ``'multiline'`` controls whether multiline support is enabled in
+      templates. For new projects this is set to ``True``, and this is the
+      recommended setting. Multiline support in templates means that newline
+      between ``{%`` and ``%}`` in templates is allowed.
+
+      It defaults to ``True``.
+
```

Comment:
This needs a versionchanged note
I also think it needs a deprecated note about the default change :thinking:

Reply:
Hmm.. I guess this text is wrong too. It defaults to False, but new projects get it set to True in the startproject template. I've fixed this.